TestApp/android/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,3 @@
 <resources>
-    <string name="appCenterCrashes_whenToSendCrashes" moduleConfig="true">ASK_JAVASCRIPT</string>
```

Comment:
Please keep these strings.

Reply:
Sure, sorry. These changes were generated by the update utility. We kept only necessary changes.